packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/controller.js

```diff
@@ -139,12 +139,20 @@ export default class CreateLinkedUserController {
 
   onLinkedUserClicked() {
     this.trackClick(`${TRACKING_ASSOCIATE_USER}-confirm`);
+
+    const service = this.PciStoragesUsersService;
     const { selected: user } = this.userModel.linkedMode;
+    const defaultCredential = user.s3Credentials[0];
```

Comment:
**suggestion:**
```suggestion
    const [defaultCredential] = user.s3Credentials;
```

Reply:
Right, thanks.
The fist approach seems more understandable (classic way).